content/installation/java/JavaOverview.md

```diff
@@ -8,3 +8,13 @@ The Contrast Java agent analyzes the behavior of Java applications running on J2
 
 ## How it Works
 To start analyzing an application, install the Java agent by adding it as `-javaagent` JVM parameter. After installation, continue to use the application as you normally would. (If it's a web application, just click on links, submit forms using normal data, and so forth.) The Java agent's sensors gather information about the application's security, architecture and libraries. You can see the results of the agent's analysis in the Contrast interface.
+
+## Note for Proguard Users
+
+[Proguard](https://sourceforge.net/projects/proguard/files/) includes Java
+bytecode optimization features which perform which break basic assumptions that
+runtime agents like Contrast rely on. Proguard users that wish to protect their
```

Comment:
I'm guessing that "which perform which break" is a typo.

Reply:
Fixed